Guides/Image_Formats.md

```diff
@@ -2,46 +2,44 @@
 
-It's much more rigidly standard than CUE, having been created for the CloneCD program. Due to CloneCD's ability to create generally high quality dumps, with a support for the majority of optical based copy protections, this makes CCD one of the best formats to dump copy protected games to. So while CUE is moreso intended to be accessible for more use cases, CCD is much better for making good quality disc dumps intended to either be directly emulated in a virtual drive, or burned to a disc to play.
+As opposed to CUE, which is intended to be accessible for more use cases, CCD is much more rigidly defined as the output of the CloneCD program, which creates very high-quality dumps and supports the majority of optical-based copy protections. This makes CCD one of the best formats for copy-protected games, either for direct emulation in a virtual drive or for burning to a physical disc. However,  Also, for the same reasons as CUE, CCD and CloneCD only support CDs.
 
-It's also of note that the .IMG itself should be identical to the .BIN, as they're both essentially raw copies of the disc's data. It also only supports CDs as well, for the same reasons that CUE only supports CDs. This means that in practice, a CCD image can be thought of a more rigidly defined CUE image that also supports subchannels.
+### Media Descriptor Image (v1) (.MDS/.MDF)
 
-### Media Descriptor File (v1) (.MDS/.MDF)
+The "Media Descriptor Image" format, almost always referred to as the MDS format after its "Media Descriptor Sidecar" extension, is once again similar to CUE and CCD in its basic structure. The .MDS file is similar to the .CCD file in that it contains information about the disc's structure (along with additional data) which is *not* presented in a human readable format. The .MDF ("Media Descriptor File") file is also similar to the .BIN and .IMG files, except that it can also contain the subchannel data directly, without using a .SUB file. This has pros and cons: it makes for less file clutter, but subchannel data is essentially never consistently dumped, meaning that an .MDF file that contains this data will not be usable for reliable DAT'ing (where the exact file hash needs to be consistent between dumps).
```

Comment:
The first sentence is unclear in its description of the why the name "MDS format" is used, and should be more explicit that the format is named after "Media Descriptor Sidecar" file, which has the .MDS extension.